network-api/networkapi/wagtailpages/migrations/0017_product_voting.py

```diff
@@ -10,7 +10,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('wagtailpages', '0015_auto_20201110_1957'),
-        ('wagtailpages', '0016_merge_20201117_1917.py'),
+        ('wagtailpages', '0016_merge_20201117_1917'),
```

Comment:
Odd little migration fix. Not sure why `.py` was in there, but removing it made my migration system happy.

Reply:
a new 0017 landed earlier today, so this is going to have to become 0018 =D

Reply:
Thanks for the heads up!